apps/guardian-ui/src/components/AdminHeader.tsx

```diff
@@ -40,10 +41,23 @@ interface AdminHeaderProps {
   connectionCode: string;
 }
 
-export const AdminHeader: FC<AdminHeaderProps> = ({ connectionCode }) => {
+export function AdminHeader({ connectionCode }: AdminHeaderProps) {
+  const [guardians, setGuardians] = useState<string | undefined>();
   const {
     state: { configGenParams },
   } = useSetupContext();
+  const { api } = useAppContext();
+  // will have to do polling
+  useEffect(() => {
+    async function getStatus() {
+      const { consensus } = await api.status();
+      const online = consensus ? consensus.peers_online + 1 : 1;
+      const offline = consensus ? consensus.peers_offline : 1;
```

Comment:
actually, for a 1/1 federation, we cannot be both online and offline at the same time. Plus, should we be checking `status.server` in that case? hmm